language/complexity-check-plugin/src/complexity-check.ts

```diff
@@ -0,0 +1,328 @@
+import {
+  Diagnostic,
+  DiagnosticSeverity,
+  Range,
+} from "vscode-languageserver-types";
+import { TextDocument } from "vscode-languageserver-textdocument";
+import { resolveDataRefs } from "@player-ui/player";
+import {
+  getProperty,
+  isPropertyNode,
+  AssetASTNode,
+  toRange,
+} from "@player-tools/json-language-service";
+import type {
+  ASTNode,
+  ASTVisitor,
+  PlayerLanguageService,
+  PlayerLanguageServicePlugin,
+  ViewASTNode,
+  DocumentContext,
+} from "@player-tools/json-language-service";
+
+const makeRange = (
```

Comment:
I think this exists in the LSP package, can we reuse this as an import?

Reply:
Actually I don't think this is even being used, so I can remove it.